### PR TITLE
FFM-6921: Updates FF relnotes for 786xx

### PR DESCRIPTION
**Pull request by StrawberryPad**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -13,6 +13,22 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 06, 2023
```
[the remark this answers is not in the document]
Comment (knagurski) [reply release-notes/feature-flags.md:16]: When I say _"we"_ I mean the platform as a whole, not the FF module

[the remark this answers is not in the document]
Comment (StrawberryPad) [reply release-notes/feature-flags.md:16]: Yes, the date is only a placeholder. Just staging this now so it's ready to put in the date and published when they give the "GO" signal. :-)

**Review by knagurski — APPROVED** [review]
Nice and simple 👍🏻

**Review by douglas-j-bothwell — APPROVED** [review]

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -13,6 +13,22 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+This release does not include new features.
+
+### Early access
+
+This release does not include early access features.
+
+### Fixed issues
+
+#### Feature Flags on the Harness Platform
```
Comment (douglas-j-bothwell) [inline release-notes/feature-flags.md:28]: Note sure why this sub-heading is here...I've never seen a level-4 subheading in Harness rel notes before.

**Review by douglas-j-bothwell — APPROVED** [review]

File: release-notes/feature-flags.md (hunk 2 above)
Comment (StrawberryPad) [reply release-notes/feature-flags.md:28]: yeah, i had the same thought, but it looks like it's been used to differentiate between "SDK notes" and others like "FF with Harness Platform"

Comment (KimberlyFields) [reply release-notes/feature-flags.md:28]: Yes, the FF team asked to have the SDK updates separated from the non-SDK updates.

**Comment by KimberlyFields** [comment]
@StrawberryPad I'm going to merge this since you have approval from technical and peer reviewers.

**Pull request merged**